x-pack/plugins/actions/server/builtin_action_types/lib/send_email.ts

```diff
@@ -116,14 +114,16 @@ export async function sendEmail(logger: Logger, options: SendEmailOptions): Prom
       if (tlsSettings?.certificateAuthoritiesData) {
         tlsConfig.ca = tlsSettings?.certificateAuthoritiesData;
       }
-      if (tlsSettings?.rejectUnauthorized !== undefined) {
-        tlsConfig.rejectUnauthorized = tlsSettings?.rejectUnauthorized;
-      }
 
+      const tlsSettingsFromConfig = getTLSSettingsFromConfig(
+        tlsSettings?.verificationMode,
+        tlsSettings?.rejectUnauthorized
+      );
+      const nodeTLSOptions = getNodeTLSOptions(tlsSettingsFromConfig.verificationMode);
       if (!transportConfig.tls) {
-        transportConfig.tls = tlsConfig;
+        transportConfig.tls = nodeTLSOptions;
```

Comment:
I'm not 100% sure of the intent here, but it looks like `tlsConfig` might have been accidentally removed. Did you mean to do this?

```suggest
        transportConfig.tls = { ...tlsConfig, ...nodeTLSOptions };
```

Reply:
You are right. Thank you for catching this!